e2e/tests/functional/plugins/conditionSet/conditionSet.e2e.spec.js

```diff
@@ -378,4 +378,83 @@ test.describe('Basic Condition Set Use', () => {
     await page.goto(conditionSet.url);
     await expect(outputValue).toHaveText('---');
   });
+
+  test('ConditionSet has correct outputs when test data is enabled', async ({ page }) => {
+    const exampleTelemetry = await createExampleTelemetryObject(page);
+
+    await page.getByTitle('Show selected item in tree').click();
```

Comment:
this isn't possible to do with linting, but we cannot use .getByTitle since title's aren't compatible with screen readers or on mobile

Reply:
Hmm. We're doing this everywhere in our tests. Perhaps it needs a separate PR to change it in one go?

Reply:
Actually I've just done that in this PR.

Reply:
They just added a linting rule for us to get away from this